### PR TITLE
Antoineh/readme update

### PR DESCRIPTION
**Pull request by antoine-haas**
Updated the readme.

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1328)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,90 @@
...
+If you're having trouble downloading & installing the bridge, see the [Known Issues](https://github.com/Microsoft/WinObjC#known-issues) section below and check our [FAQ](https://github.com/Microsoft/WinObjC/wiki/FAQ).
 
-# Welcome to the Windows Bridge for iOS project preview
-The Windows Bridge for iOS (also referred to as WinObjC) is a Microsoft open-source project that provides an Objective-C development environment for Visual Studio and support for iOS APIs. The bridge allows you to create Universal Windows Platform (UWP) apps that will run on many Windows devices using iOS APIs and Objective-C code alongside Windows 10 features like Cortana and Live Tiles.
+## Getting Started With the Bridge
+### Importing your Xcode Project
+When using the bridge, the first thing you'll want to do is generate a Visual Studio solution from your Xcode project:
 
+1. Open the Windows Command Prompt (type **cmd** in the Start Menu to find it) and navigate to your Xcode project directory using the **cd** command. Note that you should go to the directory containing your project *.xcworkspace* or *.xcodeproj* folder: you shouldn't be inside those folders, but one level above them.
```
Comment (ArnOmsft) [inline README.md]: I'd remove this which is confusing imo.

File: README.md [hunk 2]
```diff
@@ -1,71 +1,90 @@
...
 
-## Where to get the bridge
-Download the latest pre-built Windows Bridge for iOS SDK [here](https://aka.ms/winobjc_latest).
+  ```
+    cd C:\winobjc\samples\MyProject
+  ```
+2. Still from the Command Prompt, run the vsimporter tool from the *bin* directory of the bridge SDK you downloaded, which will generate the Visual Studio Solution:
 
+  ```
+    C:\winobjc\bin\vsimporter.exe
```
Comment (ArnOmsft) [inline README.md]: I would add that this command assumes the SDK has been unzipped in C:. Or we could say vsimporter.exe should be launched using its relative path to the directory where the Xcode project is

File: README.md (hunk 1 above)
Comment (andrejo-msft) [inline README.md]: I might suggest leaving the first sentence, but replace the second with:

**Note:** Use the directory containing your .workspace or .xcodeproj folder.

Sometimes too much direction is more confusing than too little.

File: README.md [hunk 3]
```diff
@@ -1,71 +1,90 @@
-###Download the latest pre-built Windows Bridge for iOS SDK [here](https://aka.ms/winobjc_latest).
+# Welcome to the Windows Bridge for iOS Project
+The Windows Bridge for iOS (also referred to as WinObjC) is a Microsoft open-source project that provides an Objective-C development environment for Visual Studio and support for iOS APIs. The bridge allows you to create Universal Windows Platform (UWP) apps that will run on many Windows devices by re-using your Objective-C code and iOS APIs alongside Windows 10 features like Cortana and Live Tiles.
```
Comment (andrejo-msft) [inline README.md]: r/Live Tiles/Windows Notifications

Overall, WNS is more easily used by the external developer. You pretty much need a WinPhone to see a live tile in action.

File: README.md [hunk 4]
```diff
@@ -1,71 +1,90 @@
...
 
-**Note on installing the bridge:** We use [Git LFS](https://git-lfs.github.com) to handle the large binaries included in the bridge. This does not affect using the [prebuilt releases found on our releases page](https://github.com/Microsoft/WinObjC/releases). If you build the bridge from source, please see the instructions found on our [Building From Source](https://github.com/Microsoft/WinObjC/wiki/Building-From-Source) wiki page.
+## Download and Installation
+To use the bridge, you'll need:
+- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest) (click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice when complete).
+- **Windows 10**, build 10586 or higher (validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system)).
+- **Visual Studio 2015 Update 3** with Windows developer tools. Visual Studio 2015 Community is available for free [here](https://dev.windows.com/downloads). Select (at least) the following components during installation:
```
Comment (andrejo-msft) [inline README.md]: Remove parenthesis.

File: README.md (hunk 2 above)
Comment (andrejo-msft) [inline README.md]: You could simply say '[location of unzipped SDK]\bin\vsimporter.exe'

File: README.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
+The Windows Bridge for iOS (also referred to as WinObjC) is a Microsoft open-source project that provides an Objective-C development environment for Visual Studio and support for iOS APIs. The bridge allows you to create Universal Windows Platform (UWP) apps that will run on many Windows devices by re-using your Objective-C code and iOS APIs alongside Windows 10 features like Cortana and Windows Notifications.
 
 ## Download and Installation
-To use the bridge, you'll need:
-- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest) (click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice when complete).
-- **Windows 10**, build 10586 or higher (validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system)).
+To use the bridge you'll need:
+- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest). Click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice when complete.
```
Comment (andrejo-msft) [inline README.md]: Suggestion: point the user to a specific landing directory as an example so you can use it when showing your command sample.

...local directory of your choice (example: **c:\winobjc**) when complete.

File: README.md [hunk 6]
```diff
@@ -17,20 +17,20 @@ If you're having trouble downloading & installing the bridge, see the [Known Iss
 ### Importing your Xcode Project
 When using the bridge, the first thing you'll want to do is generate a Visual Studio solution from your Xcode project:
 
-1. Open the Windows Command Prompt (type **cmd** in the Start Menu to find it) and navigate to your Xcode project directory using the **cd** command. Note that you should go to the directory containing your project *.xcworkspace* or *.xcodeproj* folder: you shouldn't be inside those folders, but one level above them.
+1. Open the Windows Command Prompt (type **cmd** in the Start Menu to find it) and navigate to your Xcode project directory using the **cd** command. Note: use the directory containing your .workspace or .xcodeproj folder.
 
   ```
-    cd C:\winobjc\samples\MyProject
+    [location of unzipped SDK]\MyProject> cd [location of unzipped SDK]\winobjc\samples\MyProject
```
Comment (andrejo-msft) [inline README.md]: Wrong location for prompt. This suggests you start in a non-existent directory in the SDK.

Suggest:
**c:\> cd c:\myObjCProject**

**Pull request merged**